packages/familie-endringslogg/src/chevron-lenke/chevron-lenke.tsx

```diff
@@ -27,7 +27,7 @@ const ChevronLenke = (props: ChevronLenkeProps) => {
                     <span className={'chevron-lenke__tekst'} role="button">
                         {tekst}
                     </span>
-                    <Right
```

Comment:
Stemmer det at du vil bytte ArrowLeft med Right og ArrowRight litt lenger nede med Left? Eller skulle det vært omvendt? :)

Reply:
Hvis jeg vil ha bugs; ja - ellers nei 😂
Takk for headsup!